### DISCUSSION TITLE
fastjson漏洞整改

### DISCUSSION THREAD

**Discussion by coolqz** [apache/rocketmq#7800] [General]
rocektmq集群使用二进制包v4.9.6版本部署的，先漏洞扫描出fastjson组件包1.2.69存在高危漏洞，现在要修复这个，是只能将rocketmq版本升级到5.X版本吗

**Comment by leizhiyuan** [comment]
你可以自己升级 到 [1.2.83](https://mvnrepository.com/artifact/com.alibaba/fastjson/1.2.83)  这个没漏洞的版本

**Reply by coolqz** [reply]
是直接将rocketmq安装目录下lib目录的fastjson的jar包替换成1.2.83的，然后重启broker和namesrv就行了吗

**Reply by leizhiyuan** [reply]
你如果是直接把开源的包拿去部署，不是自己编译，是可以这么做的，吧lib替换一下就可以。线下环境验证下就行。

**Reply by coolqz** [reply]
好的，谢谢
